app/scenes/ProductForm.js

```diff
@@ -3,10 +3,17 @@ import { Actions } from 'react-native-router-flux';
 
 import ProductFormView from '../components/ProductFormView';
 import { requestCreateProduct } from '../reducers/products';
+import { categoriesToObject } from '../lib/helpers';
+import { requestCategories } from '../reducers/categories';
+
+const mapStateToProps = state => ({
+  categories: categoriesToObject(state.categories.list.records),
```

Comment:
yeah, but we do that to have up-to-date list each time user visits some scene. though it's more useful in general case. we know that category list is not very likely to change.
do you want to fetch it one time when app starts or something like that?